src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -105,7 +132,7 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
 
   useEffect(() => {
     if (shouldOpenPaymentModal && !isAuthenticationModalOpen) {
-      setIsPaymentModalOpen(true);
+      setPaymentModalOpen(true);
       setShouldOpenPaymentModal(false);
     }
   }, [shouldOpenPaymentModal, isAuthenticationModalOpen]);
```

Comment:
This block of code feels convoluted. Can we just roll the logic into the open/close `useCallback`s above instead?

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1148/commits/5e9e39355ce983ca1db81c1af798117439fbca80